wrappers/python/tests/TestOpcWater.py

```diff
@@ -0,0 +1,79 @@
+import sys
+import unittest
+import openmm as mm
+from openmm import app
+from openmm import unit
+
+
+class TestOpcWater(unittest.TestCase):
+
+    def test_OpcEnergy(self):
+        pdb = app.PDBFile('../openmm/app/data/opcbox.pdb')
```

Comment:
Data files used in test cases should go in `wrappers/python/tests/systems`.  Inside the test case, you can refer to it as `systems/opcbox.pdb`.

Reply:
@peastman These are equilibrated boxes that may be useful when adding water in modeller, not just as a test case; analogous to openmm/app/data/tip3p.pdb and tip4pew.pdb etc.  I wasn't sure how to use stuff outside of tests/ in a test, but it seems like a relative path would work?

Reply:
We can always move them in the future if we decide to use them for something else.  Since the test case is the only thing they're currently used for, let's keep them with the test.

Reply:
@peastman Okay, done